customer-terminate-service/domain/src/main/java/se/inera/intyg/cts/domain/service/EraseDataForCareProvider.java

```diff
@@ -0,0 +1,97 @@
+package se.inera.intyg.cts.domain.service;
+
+import java.util.List;
+import java.util.function.Consumer;
+import java.util.function.Predicate;
+import java.util.stream.Collectors;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import se.inera.intyg.cts.domain.model.EraseService;
+import se.inera.intyg.cts.domain.model.Termination;
+import se.inera.intyg.cts.domain.model.TerminationStatus;
+import se.inera.intyg.cts.domain.repository.CertificateBatchRepository;
+import se.inera.intyg.cts.domain.repository.TerminationRepository;
+
+public class EraseDataForCareProvider {
```

Comment:
Missing @service annotation?

Reply:
This is in the domain package and should not include any spring stuff. The bean is handled in the AppConfig.

Reply:
It seems like it is missing an interface if it is to be used as a service?

Reply:
Extracted interfaces for domain services that only had implementation classes.